opwen_email_server/api/client_write.py

```diff
@@ -4,14 +4,11 @@
 from opwen_email_server import config
 from opwen_email_server import events
 from opwen_email_server.services.auth import AzureAuth
-from opwen_email_server.services.queue import AzureQueue
 from opwen_email_server.services.storage import AzureTextStorage
 from opwen_email_server.utils.log import LogMixin
 
-QUEUE = AzureQueue(namespace=config.QUEUES_NAMESPACE,
-                   sas_key=config.QUEUES_SAS_KEY,
-                   sas_name=config.QUEUES_SAS_NAME,
-                   name=constants.QUEUE_CLIENT_PACKAGE)
+import celery
```

Comment:
This import looks unused. Is it possible to remove it?

Reply:
Oops, forgot to remove this one. Will be removed.